File: 2833-furthest-point-from-origin/2833-furthest-point-from-origin.py
class Solution(object):
    def furthestDistanceFromOrigin(self, moves):
        lef=0
        rig=0
        mid=0
        for i in moves:
            if i=="L":
                lef+=1
            elif i=="R":
                rig+=1
            else:
                mid+=1
        return abs(lef-rig)+mid

       
        

File: 2833-furthest-point-from-origin/2833-furthest-point-from-origin.js
/**
 * @param {string} moves
 * @return {number}
 */
var furthestDistanceFromOrigin = function(moves) {
    let lef = 0;
    let rig = 0;
    let mid = 0;

    for (let i of moves) {
        if (i === "L") {
            lef++;
        } else if (i === "R") {
            rig++;
        } else {
            mid++;
        }
    }

    return Math.abs(lef - rig) + mid;
};